vk/backend/backend.go

```diff
@@ -101,6 +139,11 @@ func RunWithBackend(ctx context.Context, runner *runner.Runner, statuses *os.Fil
 		return errors.Wrap(err, "Could not deserialize protobuf")
 	}
 
+	useBytes, err := useByteUnits(pod)
```

Comment:
Won't this error if the annotation is missing? Do you want this call to return `numBytes, useBytes, err`?

Reply:
Doh. My bad.